docs/guides/preparing/local-server-setup.mdx

```diff
@@ -0,0 +1,302 @@
+You can now join the server from your Minecraft client at `localhost` or `127.0.0.1:25565`.
+
+Also see [Additional Setup](#additional-setup) if you would like to make the most use out of your newly-created server.
+
+<div className="text--center">
+  <ImageSwitcher loading="lazy"
+    lightImageSrc="/img/guides/local-server-setup/mac-serversetup-c1-light.png"
+    darkImageSrc="/img/guides/local-server-setup/mac-serversetup-c1-dark.png"
+  />
+</div>
+
+## Linux
+
+This guide uses commands that are available for Debian-based distributions.
+Please keep in mind that the following process may differ depending on your distribution. If necessary, consult your distribution's documentation or user manual.
+Otherwise, we trust that you know what you are doing.
+
+### Installing Java
+
+We recommend using either Java 8 or Java 11. For this guide, we will be using [OpenJDK](https://openjdk.org/). 
+However, you may choose to use [Amazon Corretto 11](https://aws.amazon.com/corretto/) or [Eclipse Adoptium](https://adoptium.net/) as well.
+If you already have a copy of Java 8 or Java 11 installed, you may skip this step.
+
+```bash
+sudo apt update && sudo apt install openjdk-11-jdk
+```
+
+### Downloading SportPaper
+
+Get the latest version of SportPaper from the [Downloads](/downloads) page.
+Once the download is done, you must put `SportPaper.jar` inside an empty folder.
+
+Next, you must create a shell script file to be able to start your server and have access to its console. 
+Start by opening your preferred text editor, copy and paste the following example below, and save the file as `start.sh`.
+
+```bash title="start.sh"
+#!/bin/sh
+
+java -Xms128M -Xmx2G -Dterminal.jline=false -Dterminal.ansi=true -jar SportPaper.jar --nogui
+```
+
+You also need to mark the script file as executable. Start by typing the following command below into your terminal.
+
+```bash
+chmod +x <path to your start.sh file>
+```
+
+You must always execute from `start.sh` to launch the server, not the .jar file.
+
+### Downloading PGM
+
+You have two options when it comes to downloading PGM. You can download the stable version of PGM from the [Downloads](/downloads) page or you can download the development builds from its [GitHub repository](https://github.com/PGMDev/PGM/actions?query=actor%3AElectroid+branch%3Adev+is%3Asuccess).
+If you are working on a map that will utilize features introduced in pre-release versions of PGM, we recommend downloading the development build.
+
+Once the download is done, you must put `PGM.jar` inside the `plugins` folder (create if it does not exist), which can be found in the folder that contains the `SportPaper.jar` file.
+
+### Starting the Server
+
+Once you are done with the initial setup, you can launch from your terminal (`./start.sh`) or double-click `start.sh` (if using a desktop environment) to let SportPaper create the necessary files for Minecraft and PGM.
```

Comment:
Double clicking start.sh might make it run, but it will not open it in a terminal window on Linux. I'd rather suggest creating an alias in `.bash_aliases` or `.bashrc`.